src/main/java/com/fasterxml/jackson/databind/jsonschema/JsonSerializableSchema.java

```diff
@@ -10,61 +10,64 @@
 /**
  * Annotation that can be used to define JSON Schema definition for
  * the annotated class.
- *<p>
+ * <p>
  * Note that annotation is often not needed: for example, regular
  * Jackson beans that Jackson can introspect can be used without
  * annotations, to produce JSON schema definition.
- * 
+ *
  * @author Ryan Heaton
  * @author Tatu Saloranta
+ * @deprecated Since 2.2, we recommend use of external
```

Comment:
Not true; was not deprecated in 2.2; this will be against 2.15

Reply:
I copied this javadoc from `com.fasterxml.jackson.databind.jsonschema.JsonSchema`. I would have thought that `JsonSerializableSchema` and `SchemaAware` were deprecated since 2.2 along with `JsonSchema` but just forgot to mark with deprecated. So It should be against to 2.15?
https://github.com/FasterXML/jackson-databind/blob/2.15/src/main/java/com/fasterxml/jackson/databind/jsonschema/JsonSchema.java#L19

Reply:
you can't just pretend this was deprecated back in v2.2 - you have to say 2.15

Reply:
Right, @luozhenyu it was not declared as deprecated and we cannot "backdate" decision -- even if that might have been intent. I suspect it was not deprecated just to reduce noise but either way `2.15` is needed here.

Reply:
Thanks, I have changed it to 2.15